charts/alfresco-search-enterprise/templates/_helpers.tpl

```diff
@@ -40,6 +40,7 @@ helm.sh/chart: {{ include "alfresco-search-enterprise.chart" . }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
+app.kubernetes.io/component: {{ include "alfresco-search-enterprise.name" . }}
```

Comment:
at this point we probably want to avoid using this named template here and just use `.Chart.Name`